### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/kubernetes-csi/docs/issues/533

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Please feel free to send PRs for the documentation in the `book` directory. If y
 We have full documentation on how to get started contributing here:
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
-- [Kubernetes Contributor Guide](http://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
+- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
```
Comment (xing-yang) [inline CONTRIBUTING.md:15]: Missing this one: http://git.k8s.io/community/contributors/guide#contributing

Comment (Arhell) [reply CONTRIBUTING.md:15]: Thanks, I'll fix it soon

**Comment by Arhell** [comment]
Done

**Pull request merged**